ckanext/stcndm/schemas/imdb.yaml

```diff
@@ -0,0 +1,185 @@
+scheming_version: 1
+about: 'Statistics Canada Dissemination: IMDB Sources'
+about_url: http://github.com/open-data/ckanext-stcndm
+catalog_type_label:
+  en: IMDB Source
+  fr: Source IMDB
+dataset_type: subject
+form_languages:
+- en
+- fr
+languages_label:
+- eng; CAN
+- fra; CAN
+
+dataset_fields:
+
+- field_name: archive_date
+  label:
+    en: Archive Date
+    fr: Date d'archivage
+  preset: date
+  form_snippet: date.html
+  display_snippet: date.html
```

Comment:
These display_snippet and form_snippet values should already be provided by the date preset.